pandas/_libs/tslibs/period.pyx

```diff
@@ -322,6 +376,19 @@ cdef int dInfoCalc_SetFromAbsDate(date_info *dinfo, int64_t absdate) nogil:
 cdef int dInfoCalc_SetFromAbsTime(date_info *dinfo, double abstime) nogil:
     """
     Sets the time part of the DateTime object.
+
+    Parameters
+    ----------
+    dinfo : date_info*
+    abstime : double
```

Comment:
can you be more specific here , e.g. what is abstime

Reply:
here

Reply:
The next line `seconds elapsed since beginning of day described by absdate` isn't clear?